db/migrate/20160429133701_create_games.rb

```diff
@@ -0,0 +1,11 @@
+class CreateGames < ActiveRecord::Migration
+  def change
+    create_table :games do |t|
+      t.string :status
+      t.integer :moves
+      t.text :board
```

Comment:
ah ok as i suspected, `postgres` can handle `json` natively so that we don't have to serialize json back and forth in the app layer. 

http://edgeguides.rubyonrails.org/active_record_postgresql.html#json

looks like a good read on the diff between json and hstore : http://www.craigkerstiens.com/2013/07/03/hstore-vs-json/